src/theme/dojo/combobox.m.css

```diff
@@ -10,18 +10,14 @@
 	width: 100%;
 }
 
-.root * {
-	box-sizing: border-box;
-}
-
-.root input {
+.root .input {
 	min-width: 200px;
 	padding: var(--spacing-regular) calc(var(--spacing-regular) * 6) var(--spacing-regular)
 		var(--spacing-regular);
 	width: 100%;
 }
 
-.clearable input {
+.clearable .input {
```

Comment:
do you think this might be better implemented / fixed using the `trailing` attribute and a click handler?

Reply:
Hm not sure what you mean here, I added a classes to the input element so the dojo styles can be added. Could you explain your suggestion further?